themes/morphosource/views/MyProjects/Media/form_html.php

```diff
@@ -78,11 +81,119 @@
 	).click(function() { this.select(); });
 	jQuery(document).ready(function(){
 		jQuery('#mediaForm').hide();
+		jQuery('.fileinput-button').hide(); 
 	});
 </script>
+<?php
+}// end if not a form post
+?>
+
+<!-- 1st form --->
+<div class="jr-container">
+    <div class="jr-group">
 
+        <div class="jfu-container" style="float:none !important">
+            <!-- The file upload form used as target for the file upload widget -->
+            <form id="fileupload" method="POST" enctype="multipart/form-data">
+                <!-- The fileupload-buttonbar contains buttons to add/delete files and start/cancel the upload -->
+                <div class="row fileupload-buttonbar">
+                    <div class="col-sm-5">
+                        <!-- The fileinput-button span is used to style the file input field as button -->
+                        <span class="btn btn-success fileinput-button fileinput-button-on-multiple button buttonMedium">
+                            <i class="glyphicon glyphicon-plus"></i>
+                            <span>Upload from computer</span>
+                            <input id="jfu-file-select_0" class="jfu-file-select" type="file" name="files[]" data-pattern-id="jfu-file-select_++">
+                        </span>
+                        <!-- The global file processing state -->
+                        <!--span class="fileupload-process"></span-->
+                        <!-- The table listing the files available for upload/download -->
+                        <table role="presentation" class="jfu-presentation table table-striped"><tbody class="files"><input type="hidden" id="presentation_0" class="presentation" data-pattern-id="presentation_++"></tbody></table>
+                    </div>
+                    <!-- The global progress state -->
+                    <div class="col-sm-5 fileupload-progress fade">
+                        <!-- The global progress bar -->
+                        <div class="progress progress-striped active" role="progressbar" aria-valuemin="0" aria-valuemax="100">
+                            <div class="progress-bar progress-bar-success" style="width:0%;"></div>
+                        </div>
+                        <!-- The extended global progress state -->
+                        <div class="progress-extended">&nbsp;</div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        
+        <button id="jr-btnRemove_0" type="button" class="jr-btnRemove jfu-hide" data-pattern-id="jr-btnRemove_++"> - </button>
+    </div>
+    <button id="jr-btnAdd" type="button" class="jr-btnAdd jfu-hide"> + </button>
+</div>
+<!-- The template to display files available for upload -->
+<script id="template-upload" type="text/x-tmpl">
```

Comment:
Are these mid-page <script> sections in the middle of the PHP needed for the jQuery file upload plugin? This is an earnest question, I'm just curious. :)

Reply:
yes, the plugin uses jquery template to render the upload progress and buttons.  I don't want to move it to the header since it's not needed on every page.